src/plusUtils.ts

```diff
@@ -80,3 +93,28 @@ export function turnOffPlus(): void {
     new CopilotPlusExpiredModal(app).open();
   }
 }
+
+export async function getComposerSystemPrompt(): Promise<string> {
```

Comment:
I suggest move anything composer related to its own file. `plusUtil` should only take care of plus user status checks